src/components/common/ModalComponent.jsx

```diff
@@ -77,7 +80,7 @@ ArtModal.defaultProps = {
   modalTitle: '',
   modalSize: 'small',
   open: false,
-  closeOnEscape: true,
-  closeOnDimmerClick: true,
+  closeOnEscape: false,
+  closeOnDimmerClick: false,
```

Comment:
I hope you have tested the change that introduced the `true` value for these...I think it was the session expired modal

Reply:
I had changed it to test something out then forgot to revert the values.